### ISSUE TITLE
I can't seem to build it , could you please provide apk files ?

### ISSUE THREAD

**Issue by SerJaimeLannister** [heroslender/lg-remote#1]
I am on archlinux and am complete noobie in kotlin * What went wrong:
Could not determine the dependencies of task ':app:lintVitalReportRelease'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     platforms;android-34 Android SDK Platform 34
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
  All licenses can be accepted using the sdkmanager command line tool:
  sdkmanager.bat --licenses
  Or, to transfer the license agreements from one workstation to another, see https://developer.android.com/studio/intro/update.html#download-with-gradle
  
  Using Android SDK: /opt/android-sdk

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

please create release app , I need lg remote app for my father for whom I installed an lg remote app from play store but its filled to ads / paywalled and its aching my stomache

**Comment by heroslender** [comment]
I'm sorry, but I just created the project and am still trying to get a starting point to get this to work, so there's no apk available yet. And yeah, those apps filled with ads are the reason i'm trying to do this, they're unusable

**Comment by SerJaimeLannister** [comment]
Agreed ! Hope you succeed in this project. People are paying 20 $ lifetime for a remote app (quite shocking) 

I was wondering if https://github.com/MaatheusGois/Remote-Control-LGTV?tab=readme-ov-file could somehow be converted to android (its electron but unfortunately electron can't be converted to android directly but there is https://nodejs-mobile.github.io/ but I am not sure)

Also I don't know kotlin and so I can't help you except by starring this project which I did , please inform me when you provide build instructions / project being decently complete.

**Comment by SerJaimeLannister** [comment]
Wishing you luck on this project !

**Comment by heroslender** [comment]
Managed to get some basic stuff to work, also released the apk in the [Releases](https://github.com/heroslender/lg-remote/releases) tab.
To connect to your tv click on `Device Disconnected` on the top and a popup with all TVs found will appear.

![screenshot](https://github.com/user-attachments/assets/8629265a-0fd9-431f-a642-45579826d2e6)

**Comment by SerJaimeLannister** [comment]
great job , I am going to try it out today. Please upload this on f-droid as well .
